app/cafe.py

```diff
@@ -0,0 +1,22 @@
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+import datetime
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+        elif visitor.get("vaccine").get("expiration_date") \
+                < datetime.date.today():

```

Comment:
I won't reject it because it won't produce any errors in this case.

But indeed, we can do visitor['vaccine'] because we have checked the presence of vaccine key on the previous step
visitor.get("vaccine").get("expiration_date") doing like this in other cases can lead to exceptions.
What if visitor.get("vaccine") returns None? We will get an error: None type has no attribute .get
It is safer to do like this:
visitor.get("vaccine", {}).get("expiration_date")